README.adoc

```diff
@@ -104,7 +104,7 @@ b. log in as "developer".
 +
 [source,subs="quotes"]
 ----
-oc new-project _project_name_
+oc new-project _project-name_
```

Comment:
It's a placeholder, not an example value. The name of the placeholder is not relevant. But obviously, it's confusing so let's improve it like this:
```
----
oc new-project _PROJECT_NAME_
----
+
where _PROJECT_NAME_ is the name of the project.
```
Alternative placeholder formats:
1. `PROJECT_NAME`,
2. `<PROJECT_NAME>`,
3. `$PROJECT_NAME`,
4. `${PROJECT_NAME}.

Priority number one is consistency across the documentation.

- I don't like 3 and 4 because it collides with an actual usage of an env variable in the command.
- I insist on the underscore. Best readability. Seems to be very common approach.
- I like all caps. Makes it obvious it's not a program name but some kind of variable. But I don't insist. CRC uses lower case.
- Angle brackets in 2 may collide with shell redirect operator but make it obvious it's a placeholder. CRC, Quarkus, and OpensShift uses that. Angle brackets definitely collide in XML context but not in JSON.
- +1 for italics style. Makes it stand out in the rest of the source code. CRC uses that.

https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_cli/getting-started-cli.html
https://code-ready.github.io/crc/
https://quarkus.io/guides/deploying-to-kubernetes

@emmurphy1 any advice?